packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -183,16 +183,18 @@ module.exports = class AwsS3Multipart extends Plugin {
         onPartComplete: (part) => {
           // Store completed parts in state.
           const cFile = this.uppy.getFile(file.id)
-          this.uppy.setFileState(file.id, {
-            s3Multipart: Object.assign({}, cFile.s3Multipart, {
-              parts: [
-                ...cFile.s3Multipart.parts,
-                part
-              ]
+          if (cFile) {
```

Comment:
Thanks! Maybe we could return early here, something like `if (!cFile) return`, then we don’t need deep nesting.